setup.py

```diff
@@ -153,7 +153,9 @@
             ('generic_object_definitions = '
                 'cfme.generic_objects.definition:GenericObjectDefinitionCollection'),
             ('generic_objects = '
-                'cfme.generic_objects.instance:GenericObjectInstanceCollection')
+                'cfme.generic_objects.instance:GenericObjectInstanceCollection'),
+            ('orchestration_template = '
```

Comment:
**REQUIRED**
Please plural form in collections names.

Reply:
:+1: